.s2i/action_hooks/build

```diff
@@ -38,6 +38,11 @@ while read line; do
     esac
 done < requirements-optional.txt
 
+# For openshift deployment we will also install all requirements for
+# social-auth-core package, otherwise OpenID providers would not work
+
+pip install 'social-auth-core[openidconnect]>=3.1.0'
```

Comment:
@nijel so i removed this from `requirements.txt` and added it directly here. Would that work for you?